Airports/EGLF/SMR/Geo.txt

```diff
@@ -1,202 +1,709 @@
-N051.16.57.413 W000.46.27.429 N051.16.56.164 W000.46.28.086 smrWhite ;ApronSplit (TG.B)
 
+; - Building_seperator
+N051.16.43.155 W000.46.54.459 N051.16.41.427 W000.46.50.648 smrWhite
+; - Building_seperator
+N051.16.45.651 W000.46.51.608 N051.16.43.939 W000.46.47.724 smrWhite
+; - Building_seperator
+N051.16.53.366 W000.46.46.781 N051.16.51.446 W000.46.42.644 smrWhite
+; - Building_seperator
+N051.16.55.711 W000.46.44.059 N051.16.53.836 W000.46.39.973 smrWhite
+; - Apron_seperator
+N051.16.47.767 W000.46.46.126 N051.16.47.185 W000.46.44.783 smrWhite
+; - Apron_seperator_1
+N051.17.01.353 W000.46.30.326 N051.16.56.782 W000.46.32.731 smrWhite
+; - Apron_seperator_2
+N051.17.00.509 W000.46.28.330 N051.16.56.337 W000.46.30.486 smrWhite
+; - Apron_seperator_3
+N051.16.57.080 W000.46.27.567 N051.16.55.776 W000.46.28.239 smrWhite
+; - Building_seperator
+N051.16.46.394 W000.46.23.585 N051.16.48.738 W000.46.25.654 smrWhite
+; - Building_seperator
+N051.16.47.469 W000.46.20.335 N051.16.49.750 W000.46.22.445 smrWhite
+; - Building_seperator
+N051.16.48.554 W000.46.17.048 N051.16.50.880 W000.46.19.119 smrWhite
+
+; - G1
+N051.16.55.454 W000.45.38.450 N051.16.54.954 W000.45.38.438 smrWhite
+; - F1
+N051.16.17.763 W000.47.30.676 N051.16.17.552 W000.47.29.644 smrWhite
+; - F2
+N051.16.22.345 W000.47.14.861 N051.16.22.964 W000.47.12.989 smrWhite
+; - F3
+N051.16.28.476 W000.47.07.254 N051.16.28.084 W000.47.06.549 smrWhite
+; - F4
+N051.16.33.770 W000.46.59.840 N051.16.33.435 W000.46.59.207 smrWhite
+; - E
+N051.16.31.570 W000.46.51.423 N051.16.32.009 W000.46.50.538 smrWhite
+; - B6
+N051.16.54.078 W000.46.38.596 N051.16.53.656 W000.46.37.558 smrWhite
+; - D4
+N051.16.52.829 W000.46.30.950 N051.16.52.890 W000.46.29.680 smrWhite
+; - D3
+N051.16.50.707 W000.46.30.749 N051.16.50.775 W000.46.29.408 smrWhite
+; - D2
+N051.16.47.349 W000.46.30.227 N051.16.47.400 W000.46.28.854 smrWhite
+; - D1
+N051.16.38.779 W000.46.29.133 N051.16.39.631 W000.46.26.627 smrWhite
+; - J1
+N051.16.49.382 W000.46.27.304 N051.16.48.871 W000.46.26.871 smrWhite
+; - J2
+N051.16.51.638 W000.46.20.284 N051.16.51.239 W000.46.19.898 smrWhite
+; - B5
+N051.16.53.769 W000.46.23.316 N051.16.54.170 W000.46.23.111 smrWhite
+; - B4
+N051.16.52.037 W000.46.15.163 N051.16.52.536 W000.46.14.906 smrWhite
+; - B3
+N051.16.51.285 W000.46.11.298 N051.16.51.762 W000.46.11.063 smrWhite
+; - B1
+N051.16.48.584 W000.46.00.442 N051.16.49.916 W000.45.59.749 smrWhite
+; - C
+N051.16.47.929 W000.46.06.734 N051.16.47.155 W000.46.04.869 smrWhite
+; - A2
+N051.16.55.779 W000.45.53.616 N051.16.55.313 W000.45.53.653 smrWhite
+; - A1
+N051.16.54.322 W000.45.41.514 N051.16.54.293 W000.45.40.380 smrWhite
+; - G2
+N051.16.56.274 W000.45.49.906 N051.16.56.242 W000.45.49.102 smrWhite
+; - Z
+N051.16.20.602 W000.46.59.825 N051.16.19.958 W000.46.59.461 smrWhite
+; - Y
+N051.16.26.446 W000.46.42.314 N051.16.26.118 W000.46.42.926 smrWhite
+; - W
+N051.16.30.856 W000.46.27.163 N051.16.30.911 W000.46.26.057 smrWhite
```

Comment:
Add section title.
Put comments on end of line of singkle line items.
Put into alphanermic order.

Reply:
forgot i had to do that :)